lib/consensus.js

```diff
@@ -211,9 +211,10 @@ function _getAncestorHashes({allXs, ledgerNodeId}) {
           if(!hashes.has(parentHash) &&
             !event._parents.some(p => p.eventHash === parentHash)) {
             parentHashes.add(parentHash);
+            hashes.add(parentHash);
```

Comment:
This line changes how the algorithm works and causes regular events to not have their ancestry processed and added.

Reply:
I think I have a fix for it... working on it.